src/settings-ui/Settings.UI/SettingsXAML/OobeWindow.xaml.cs

```diff
@@ -105,6 +105,14 @@ public OobeWindow(PowerToysModules initialModule, bool isDark)
             });
         }
 
+        public void SetAppWindow(PowerToysModules module)
+        {
+            if (shellPage != null)
+            {
+                shellPage.NavigateToModule(this.initialModule);
```

Comment:
Shouldn't this be `module`, or this method parameter is redundant.

```suggestion
                shellPage.NavigateToModule(module);
```

Reply:
Good catch - that's what I get for copy/paste mindlessly. 

Fixed.